### ISSUE TITLE
How about doing top domains by h-index?

### ISSUE THREAD

**Issue by danluu** [antontarasenko/smq#3]
I see you have domains by median, and submitters by h-index. Why not domains by h-index? :-).

I imagine the list will be dominated by news sites like techcrunch and the NYC, shared hosting platforms like medium and tumblr, and community sites like wikipedia and github, but I'd still be curious to see what the results are if it's easy for you to run the query.

**Comment by antontarasenko** [comment]
Sure. In `hackernews/top-authors-h-index.sql`, I replace `[by] author` with `domain(url) author`. And here's the top 25 (verbatim):

```
github.com  279  
null    276  
google.com  259  
nytimes.com 238  
blogspot.com    234  
techcrunch.com  208  
github.io   194  
arstechnica.com 185  
medium.com  183  
wired.com   180  
tumblr.com  172  
wordpress.com   170  
washingtonpost.com  167  
mozilla.org 163  
eff.org 163  
theguardian.com 147  
theatlantic.com 143  
bbc.co.uk   143  
wsj.com 142  
wikipedia.org   132  
youtube.com 130  
amazon.com  127  
apple.com   127  
reddit.com  126  
bloomberg.com   123  
```

**Comment by danluu** [comment]
Am I misunderstanding how you're defining h-index? I thought that, for example, an h-index of 20033 would imply 20033 stories of score `>= 20033`.

**Comment by antontarasenko** [comment]
@danluu You're right. I should have replaced `[by]` twice. I fixed the previous post. Thanks!

**Comment by danluu** [comment]
Cool. Thanks! I was wondering if pg was so prolific that he'd "beat" some of these sites that have hundred of authors.
